Npm problem on branch Centos

when run ```npm --vesrsion``` :

```module.js:471
    throw err;
    ^

Error: Cannot find module 'semver'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/usr/lib/node_modules/npm/lib/utils/unsupported.js:2:14)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)```

--- a/vagrant/provisioning/ideato.nodejs/tasks/main.yml
+++ b/vagrant/provisioning/ideato.nodejs/tasks/main.yml
@@ -1,13 +1,12 @@
 ---
 
-- name: add nodejs ppa
-  shell: get_url -sL https://rpm.nodesource.com/setup_{{ nodejs_version }} | sudo -E bash -
+- name: add nodejs repo
+  shell: curl -sL https://rpm.nodesource.com/setup_{{ nodejs_version }} | sudo -E bash -
 
 - name: install nodejs
   yum: name={{ item }} state=latest
   with_items:
     - nodejs
-    - npm
 
 - name: Update NODE_PATH
   lineinfile: dest=/home/vagrant/.bashrc line='export NODE_PATH="$NODE_PATH:./node_modules"'

